### DISCUSSION TITLE
Description field for Blocks

### DISCUSSION THREAD

**Discussion by cata008** [Azure/ipam#197] [Ideas]
Hi @DCMattyG,

Could we have a description field for the blocks that we create in IPAM? It would turn out useful for situations where you have multiple blocks targeting different parts of the infra or simply BUs.

Something like:
Name  Description  Parent Space  CIDR

Thanks,
Catalin

**Comment by DCMattyG** [comment]
Hey there @cata008, this is a relatively straight-forward ask and I think it can be done without too much fuss.

I'll add this to my near-term roadmap for you.
